service/initialise.go

```diff
@@ -108,21 +108,32 @@ func (e *Init) DoGetMongoDB(ctx context.Context, cfg *config.Config) (api.MongoS
 		Collection: cfg.MongoConfig.Collection,
 		Database:   cfg.MongoConfig.Database,
 		URI:        cfg.MongoConfig.BindAddr,
+		Username:   cfg.MongoConfig.Username,
+		Password:   cfg.MongoConfig.Password,
+		IsSSL:      cfg.MongoConfig.IsSSL,
 	}
-	if err := mongodb.Init(ctx); err != nil {
+	if err := mongodb.Init(ctx, false, true); err != nil {
```

Comment:
these hard-coded values (`false, true`)
seem to be at odds with the configurability of others (`shouldEnableReadConcern, shouldEnableWriteConcern`)
can you confirm this is right?